### ISSUE TITLE
Update `ipfs-core` dependency

### ISSUE THREAD

**Issue by felladrin** [dmotz/trystero#35]
Hi 👋 Thanks for creating this lib!

After installing it with NPM, I've got a lot of warnings due to vulnerabilities found on sub-dependencies of _ipfs-core@0.9.0_. Updating it to the latest version [ipfs-core@0.18.0](https://www.npmjs.com/package/ipfs-core) resolved it.

As I'm not using the ipfs strategy, I've set an [override](https://docs.npmjs.com/cli/v9/configuring-npm/package-json?v=true#overrides) for it as a workaround, and I confirmed that using the latest version ([ipfs-core@0.18.0](https://www.npmjs.com/package/ipfs-core)) resolves all the warnings.

```json
"overrides": {
  "ipfs-core": "latest"
},
```

Could you update it in the lib dependencies, so we don't need the workaround anymore?
https://github.com/dmotz/trystero/blob/036a8e7c408b8f4946e0e8d6eb09ecf33bcc08b6/package.json#L27

**Comment by dmotz** [comment]
Yeah I've been aware of these vulnerability warnings but unfortunately the ipfs-core API has changed significantly in the newer releases and it's not as simple as a package update. I've tried a few times to get it working with the latest package, but haven't had luck — it's a very complex package. If you have any interest in giving it a shot, let me know.

I'm not sure if there's a good solution to the warning in the meantime, which is probably irrelevant to most users since I don't think many devs opt for the IPFS strategy. For what it's worth, I think the vulnerabilities are only relevant to running ipfs-core 0.9.0 on Node and not relevant to the browser.

**Comment by dmotz** [comment]
Fixed in [0.16.0](https://github.com/dmotz/trystero/releases/tag/0.16.0)
